schemas/dependencies/slack-workspace-1.yml

```diff
@@ -56,6 +56,7 @@ properties:
           - sentry-helper
           - slack-sender
           - openshift-upgrade-watcher
+          - ocm-clusters
```

Comment:
as discussed in https://github.com/app-sre/qontract-reconcile/pull/2452, it could make more sense to send these message from `ocm-upgrade-watcher`.